### DISCUSSION TITLE
Code Issue

### DISCUSSION THREAD

**Discussion by bilpiqc** [community/community#48623] [Other Feature Feedback, Questions, & Ideas]
/
![Github](https://user-images.githubusercontent.com/122728567/221401341-cd733336-275a-4d36-af85-5af5d83cc40c.PNG)
I need help regarding this code how to resolve this issue

**Comment by SmashedFrenzy16** [comment]
For the warning, put the line: `<html lang="en">` after `<!DOCTYPE html>`.

For error 1, declare character encoding with this line: `<meta charset="UTF-8">`.

For error 2, at the top of the document put this line: `<!DOCTYPE html>`.

For error 3, inside the `<head>` tags put the line: `<title>YOUR TITLE</title>`, where `YOUR TITLE` should be replaced with the title of the program.

Basically, the basic structure of a HTML document should look like this: 

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>YOUR TITLE</title>
</head>
<body>
    
</body>
</html>
```

Hope this helps!
